### DISCUSSION TITLE
[Esame 13 Febbraio 2023] Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#353] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/0319d24d-82c9-41ea-9a30-7278443cf8ef)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/e025ee05-af47-4dec-9a6a-47f16e40f1cd)
